test/unit/tarantool_role_test.lua

```diff
@@ -0,0 +1,594 @@
+local expirationd = require("expirationd")
+local fiber = require("fiber")
+local t = require("luatest")
+local helpers = require("test.helper")
+local g = t.group('tarantool_expirationd_role')
```

Comment:
Please, separate group initialization and packages require.

Reply:
changed